method-info/totals-and-components/totals-and-components.md

```diff
@@ -0,0 +1,115 @@
+# Totals and Components User Notes
+
+## Finding and Installing the Method
+You can find instructions on downloading and installing the method in the [Help centre](https://statisticalmethodslibrary.ons.gov.uk/help-centre/index) of the [ONS Statistical Methods Library](https://statisticalmethodslibrary.ons.gov.uk)
+
+## Using the Method
+
+### Overview
+Once the totals and components method is available on your computer you will be
+able to call the method on a set of data. The tcc markers returned determine
+if and what the method has corrected.
+
+* "S" is when the method stops
```

Comment:
May be worth clarifying that it stops early/method is not completed

Reply:
Done :)